src/crimson/os/seastore/seastore_types.h

```diff
@@ -49,6 +49,15 @@ constexpr segment_id_t BLOCK_REL_SEG_ID =
 // for tests which generate fake paddrs
 constexpr segment_id_t FAKE_SEG_ID =
   std::numeric_limits<segment_id_t>::max() - 4;
+using device_id_t = uint8_t;
+constexpr uint16_t device_id_len = 8;
```

Comment:
4 bits according to the initial design doc? I think we won't need 8 bits for the limited number of storage tiers per crimson OSD, and it will squeeze the addressable space out of paddr.

Reply:
oh, yes, will correct it.